src/test/java/de/retest/recheck/report/TestReportFilterTest.java

```diff
@@ -177,6 +194,23 @@ void test_replay_result_should_be_filtered_properly() throws Exception {
 		assertThat( differences ).containsExactly( notFilterMe );
 	}
 
+	@Test
+	void test_replay_result_should_keep_golden_master_exceptions() {
```

Comment:
I would argue we can remove this because we have the other test now. (Otherwise, why not have the same test for other parent containers such as `SuiteReplayResult` or `TestReport`?)